core/trino-main/src/test/java/io/trino/operator/TestExchangeClient.java

```diff
@@ -391,6 +392,80 @@ public void testStreamingAddLocation()
         assertStatus(statuses.get(location2), location2, "closed", 3, 3, 3, "not scheduled");
     }
 
+    @Test
+    public void testDeduplication()
+            throws Exception
+    {
+        DataSize maxResponseSize = DataSize.of(10, Unit.MEGABYTE);
+        MockExchangeRequestProcessor processor = new MockExchangeRequestProcessor(maxResponseSize);
+
+        TaskId taskP0A0 = new TaskId(new StageId("query", 1), 0, 0);
+        TaskId taskP1A0 = new TaskId(new StageId("query", 1), 1, 0);
+        TaskId taskP0A1 = new TaskId(new StageId("query", 1), 0, 1);
+
+        URI locationP0A0 = URI.create("http://localhost:8080/1");
+        URI locationP1A0 = URI.create("http://localhost:8080/2");
+        URI locationP0A1 = URI.create("http://localhost:8080/3");
```

Comment:
nit: I would replace `P` and `A` here with underscores

Reply:
Same question here (do we use snake case anywhere else?)

Reply:
Good point - I do not believe we do. Leave it then. Even though it is hard to read.